src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1682,8 +1685,10 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
                                      (char *)NULL)) {
     } else if (ceph_argparse_flag(args, i, "--pretty-format", (char *)NULL)) {
       cfg->pretty_format = true;
+    } else if (ceph_argparse_flag(args, i, "--legacy", (char *)NULL)) {
+      cfg->legacy = true;
     } else if (ceph_argparse_flag(args, i, "--try-netlink", (char *)NULL)) {
-      cfg->try_netlink = true;
+      *err_msg << "rbd-nbd: Nope, this is the default and it will be removed later!";
```

Comment:
Nit: this currently won't be printed unless an error code is returned.

```suggestion
      *err_msg << "rbd-nbd: --try_netlink is deprecated.";
```

Reply:
It looks like we output `err_msg` in `rbd_nbd` only if `parse_args` returns with an error. So it will be just ignored.

And @dillaman I am not sure I like very much if it would just tell "depricated". To me a user might think that netlink interface is deprecated, not the option. May be still output something  like `.. is deprecated (already default)`?

Probably we could have a message like this to the log too? If it is some third party software that calls rbd-nbd with this option, it is more likely admins would notice the message in the log than in stderr.

Reply:
Ah, sorry, @dillaman already noted that it won't be printed.

Reply:
`.. is deprecated (already default)`, I think this could give more useful info.